destral/testing.py

```diff
@@ -158,6 +158,66 @@ def test_access_rules(self):
                     ', '.join(no_access)
             ))
 
+    def test_translate_modules(self):
+        """
+        Test translated strings in the module using the .po and .pot files
+        """
+
+        from os.path import join, isdir
+        from tools import trans_export
+        from cStringIO import StringIO
+        from utils import compare_pofiles
+        mod_path = join(self.openerp.config['addons_path'], self.config['module'])
+        trad_path = join(mod_path, 'i18n')
+        if not isdir(trad_path):
+            logger.warning(
+                'Module %s has no translations', self.config['module']
+            )
+            return
+
+        logger.info(
+            'Checking translations for module %s', self.config['module']
+        )
+        with Transaction().start(self.database) as txn:
+            cursor = txn.cursor
+
+            # Generate POT data from loaded strings
+            trans_data = StringIO()
+            trans_export(
+                self.config['testing_langs'][0], self.config['module'],
+                trans_data, 'po', dbname=cursor.dbname
+            )
+
+        tmp_pot = '/tmp/{}.pot'.format(self.config['module'])
+        try:
+            # Write POT data into temp file
+            with open(tmp_pot, 'w') as pot:
+                pot.write(trans_data.getvalue())
+            pot_path = join(trad_path, '{}.pot'.format(self.config['module']))
+            missing_strings = compare_pofiles(tmp_pot, pot_path)
+
+            self.assertFalse(
+                missing_strings,
+                'There are {} missing strings in the POT file'
+                ' of the module {}'.format(
+                    missing_strings, self.config['module']
+                )
+            )
+            for test_lang in self.config['testing_langs'].split(','):
```

Comment:
you don't need to use split, if the environment lang can be parsed as a list it will be a native python list.
e.g.:

```shell
export DESTRAL_TESTING_LANGS="['es_ES', 'en_US']"
```

then you can iterate over the variable

```python
for test_lang in self.config['testing_langs']:
    ...
```